.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	extends: [
+		'eslint:recommended',
+		'plugin:react/recommended',
+		'plugin:@typescript-eslint/recommended',
+		'prettier',
+	],
+	overrides: [],
+	parser: '@typescript-eslint/parser',
+	parserOptions: {
+		ecmaVersion: 'latest',
+		sourceType: 'module',
+	},
+	plugins: ['react', '@typescript-eslint'],
+	rules: {
```

Comment:
let's add `@typescript-eslint/explicit-function-return-type` rule

Reply:
Added this in rules with 'off' and in overrides to enable this rule for .ts and .tsx files only (if this is redundant and it should be just in rules, please let me know)